TimerGitExample/TimerGitExample/ViewController.swift

```diff
@@ -14,8 +14,12 @@ class ViewController: UIViewController {
         // Do any additional setup after loading the view.
         //this is my sec commit
         // Feature branch changes
+        //
     }
 
 
 }
 
+class example {
```

Comment:
Can you make the class name in camel case

Reply:
done